### ISSUE TITLE
[Feature Request]: Option for hijacking input only when the screen is focus on.

### ISSUE THREAD

**Issue by AmyRoxwell** [PCSX2/pcsx2#10443]
### Description

On 1.6, you were able to use your controller on other windows without sending any inputs to pcsx2 unless the window was focus on, so you didn't have to worry about pcsx2 admitting inputs when you didn't want it, sadly, this is not longer the case on the newest nightly version of the emulator, now it is forced to always accept all inputs not matter what, making certain tasks more difficult or just impossible to do currently.

Examples as, having multiple windows of PCSX2 open so you can test a fan-translation or a romhack while you are developing but you don't want all of those windows to accept your controller input at the same time . Also like, testing a replacement server for a game in different version of said game, to make sure they are working or if they have issues but you don't want the inputs on one version to the other.

Or in my case, where I want to showcase something outside of pcsx2 that requires my controller to send inputs but I cannot pause pcsx2 due to me being connected online to a server and not wanting to lose connection.

### Reason

This will allow for a faster and overall, easier worflow for romhackers that want to test stuff on pcsx2 and also people that may want to have multiple pcxs2 windows for x or y reasons.

### Examples

Sadly I don't know if any emulator that uses QT and/or SDL had implemented this before, so I don't have any examples for it...
The thing that matters most is that this needs to be able to stop sending controller inputs without pausing the emulator, due to some cases where doing so would cause issues

**Comment by refractionpcsx2** [comment]
Is this only with controllers? my keyboard doesn't work in pcsx2 unless I'm focused here.

**Comment by Daniel-McCarthy** [comment]
> Is this only with controllers? my keyboard doesn't work in pcsx2 unless I'm focused here.

Yes, this is a controller thing. You can continue to use inputs in the game even when the screen is out of focus. I will add that this is quite useful, and there's both legitimate use cases for having this off as well as on, so I'm in favor of a toggle-able option for this.

**Comment by AmyRoxwell** [comment]
> Is this only with controllers? my keyboard doesn't work in pcsx2 unless I'm focused here.

Yes! this is a controller only issue, with keyboard funnily enough it doesn't happen

**Comment by stenzek** [comment]
Not sure if SDL exposes this in a cross-platform manner.

**Comment by AmyRoxwell** [comment]
> Not sure if SDL exposes this in a cross-platform manner.

Yeag, that's what's worrying me, that it's a quirk with SDL that you can't do anything about it :thinking:

**Comment by coornio** [comment]
I can see why that particular setup is an issue, but PCSX2 capturing my controller even when unfocused is handy when I'm playing/testing something and keep shifting focus to different windows to browse or note stuff down. If it ever changes, it'd be nice to be on a toggle :)
